hyperlink/_url.py

```diff
@@ -861,6 +917,8 @@ def replace(self, scheme=_UNSET, host=_UNSET, path=_UNSET, query=_UNSET,
             port=_optional(port, self.port),
             rooted=_optional(rooted, self.rooted),
             userinfo=_optional(userinfo, self.userinfo),
+            family=_optional(family, self.family),
+            uses_netloc=_optional(uses_netloc, self.uses_netloc)
```

Comment:
This seems incorrect.  You shouldn't be able to specify `uses_netloc` to the constructor at all, it's entirely a function of the scheme; adding it to `replace` seems to be propagating the error.